client/src/legacy/HtmlEditorField.js

```diff
@@ -289,7 +290,7 @@ ss.editorWrappers.tinyMCE = (function() {
       if(cb) href = eval(cb + "(href, node, true);");
 
       // Turn into relative, if set in TinyMCE config
-      if(cu && href.match(new RegExp('^' + tinyMCE.settings['document_base_url'] + '(.*)$'))) {
+      if(cu && href.match(new RegExp('^' + escapeRegExp(tinyMCE.settings['document_base_url']) + '(.*)$'))) {
```

Comment:
This isn't directly related to the _purpose_ of this PR, but I'm adding escapeRegExp in this PR so it makes sense to apply it in the one other place I found that part of a URL is used as a variable in `new RegExp()`.